backend/api/v1/endpoints/research.py

```
@@ -73,26 +75,43 @@
+    profile = db.query(User).filter(User.id == current_user.id).first()
+
+    if not profile or not profile.fields_of_interest:
+        # Return latest papers if interests not set
+        return db.query(ResearchPaper).order_by(ResearchPaper.created_at.desc()).limit(10).all()
+
+    # ✅ Fix: Split comma-separated string into clean list of interests
+    interests = [field.strip().lower() for field in profile.fields_of_interest.split(",")]
+
+    # Fetch papers matching user's interests
+    matched_papers = db.query(ResearchPaper).filter(
+        func.lower(ResearchPaper.research_field).in_(interests)
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)